### DISCUSSION TITLE
Distinction between 'forecastHourly' and 'forecast' and how to retrieve 'marine' forecasts.

### DISCUSSION THREAD

**Discussion by pbeeken** [weather-gov/api#515] [Q&A] [answered]
I am new to the weather api but familiar with REST calls in general.  I am building a custom kiosk for retrieving local marine weather forecasts for a club near the water.

 1. After some experimentation I discovered that the `...forecast` and `...forecast/hourly` urls return exactly the same information. Is this intentional?  Perhaps this is an area for future expansion?
 2. In order to check the integrity of the received information I checked the content against the website for the exact match to the gird coordinates and see different (not wildly different) reports.  Is this normal given that the website may be using older access methods?
 3. Perhaps this isn't ready yet but is there a way to access marine forecast data (tends to include things like wave action and craft warnings)? 
 4. Minor question, on the website for a given location, the current conditions are listed.  Is there a rest call to retrieve that information?

**Comment by JJKraw** [comment, marked as answer]
Can you point out a specific office and grid coordinates where forecast/ and forecast/hourly return the same information? The former returns forecast data for 12-hour periods while the latter returns one hour periods.

I do not believe marine forecasts have been implemented yet.

Current conditions can be found via stations/{station ID}/observations

Hopefully you know documentation is available at:
   https://www.weather.gov/documentation/services-web-api

**Comment by pbeeken** [comment]
Thank you for your reply. I figured the marine data might not be ready.  I am really loving the json reply structure it makes it very easy to decompose and interpret.  The documentation was helpful but I didn't realize that the `observations` was current information (nomenclature misunderstanding) I thought it was a list of historical entries.  Now that I know what it is I'll have no trouble deciphering it.  Thanks again.

The terrestrial information:
```
 'cwa': 'OKX',
 'gridId': 'OKX',
 'gridX': 39,
 'gridY': 45,
```
The marine location is from `zoneid=anz335` with map coordinates about a mile south off shore `lon=-73.73154 lat=40.89919`
